packages/node-red-s1seven-api/lib/identities/identities.js

```diff
@@ -39,12 +42,13 @@ module.exports = function (RED) {
         node.warn(RED._('identity.errors.companyId'));
         done();
       } else {
-        try {
-          const response = await axios.get(url, {
+        const { success, data } = await requestHandler(
+          get(url, {
             headers: {
               Authorization: `Bearer ${accessToken}`,
               'Content-Type': 'application/json',
               company: companyId,
+              'x-version': `${version}`,
```

Comment:
just a comment for future improvement. 
i think we should find a way to improve the `requestHandler` to allow injecting common headers into the request without having to bother in the node...

Reply:
no that will be for a future refactoring where we would also try to return response headers (and not only body), to allow handling rate limit.